requirements-dev.txt

```diff
@@ -2,6 +2,7 @@ flake8
 pytest
 vcrpy
 ansible_runner
+python-debian
```

Comment:
should that also be present for normal installs? if so, `requirements-common` is better suited.

(we don't have to make the built rpm depend on that)

Reply:
Well, it's optional. otherwise we would not need the guarding of its import.

Reply:
The guarding is an ansible convention, we do the same for apypie. But I see your point. shall we start an optional.txt? And move ipaddress there too?